readme.md

```diff
@@ -13,6 +13,12 @@
 
 ---
 
+## Requirements:
+
+`Nodejs 14`
```

Comment:
Looks good! Maybe a bullet point and no code block around "Nodejs 14" just so we have commands and code in code blocks otherwise